web/cobrands/sass/_layout.scss

```diff
@@ -821,33 +833,38 @@ textarea.form-error {
 
 // big yellow bit full screen width
 #front-main {
-  color: $primary_text;
-  background: $primary;
-  margin: 0;
+  color: $front-main-color-desktop;
+  background: $front-main-background-desktop;
   padding: 1em;
+  margin: 0 auto;
   #front-main-container {
     max-width: $container-max-width;
     margin: 0 auto;
+    background: $front-main-container-background-desktop;
+    @if not (($front-main-container-background-desktop == transparent) or ($front-main-container-background-desktop == none)) {
+      padding: 1em;
+    }
   }
   h2 {
     font-style:normal;
     margin:0;
-    color: inherit;
-  }
-  #postcodeForm {
-    color: inherit;
-    background:none;
-    overflow:hidden;
-    padding-bottom: 0;
-    label {
-      margin:0.5em 0;
-    }
-    .form-hint {
-        margin-top: -0.25em;
-    }
-    div {
-      width:20em;
-    }
+    color: $front-main-h2-color-desktop;
+  }
+}
+
+.postcode-form-box {
+  color: $front-main-color-desktop;
+  background:$postcodeform-background-desktop;
+  overflow:hidden;
+  label {
+    margin:0.5em 0;
+  }
+  .form-hint {
+      color: $form-hint-color-desktop;
+      margin-top: -0.25em;
```

Comment:
This isn't matching, same reason as what I've just outlined about base margin. Again, I think a `#front-main p.form-hint` would be easiest/clearest here, probably?

Reply:
sorry not sure I get this one. If I change the colour using the variable it changes in the homepage and alert page.

Reply:
lol, sorry. It makes sense I'll add a commit now